src/App.js

```diff
@@ -191,7 +191,7 @@ const returnStartedRoute = (Component, props, rest) => {
   if (hathorLib.wallet.loaded()) {
     // Wallet is locked, go to locked screen
     if (hathorLib.wallet.isLocked()) {
-      return <Redirect to={{pathname: '/locked/'}}/>;
+      return <Redirect to={{pathname: '/locked/', hash: rest.location.hash}}/>;
```

Comment:
Before redirecting to `/wallet_type/#ledger:closed` we lock the wallet and this is the actual route taken.
Below the `returnDefaultComponent` method will intercept the `/locked/` route and redirect back to `/wallet_type/` for hardware wallets, we need to pass the hash in both instances so that it reaches the actual wallet type screen.

Reply:
Outdated